### DISCUSSION TITLE
vim-rhubarb for helix

### DISCUSSION THREAD

**Discussion by brycefisher** [helix-editor/helix#7323] [Ideas]
I regularly navigate to code in my editor, then want to share that exact selection with my coworkers on slack. When I was using vim, I used the [vim-rhubarb](https://github.com/tpope/vim-rhubarb) plugin so that I get a keymap binding to yank the current selection's github URL into my clipboard. That's one thing I still miss from vim in my daily workflow (not a whole lot else TBH).

Its really hard to search for this on github, but I can't seem to find any similar feature request discussion already around.

**Comment by kirawi** [comment]
Could probably use https://github.com/helix-editor/helix/pull/6979
